server/src/constants/settings/interfaces/settings.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { FileFolder } from 'src/core/file/interfaces/file-folder.interface';
+import { ShortCropSize } from 'src/utils/image';
+
+type ValueOfFileFolder = `${FileFolder}`;
+
+export interface Settings {
+  storage: {
+    imageCropSizes: {
+      [key in ValueOfFileFolder]: ShortCropSize[];
```

Comment:
we are adding a dependency between settings and files module but I believe it's in the right way so LGTM